Assets/AishaBikebayeva/Arcade Game/Scripts/boat.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class boat : MonoBehaviour
+{
+    public float speed = 1;//0.0f;
+    public Rigidbody2D rb;
+    public Vector2 movement;
+ 
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb = this.GetComponent<Rigidbody2D>();
+
+        //    rb.velocity = new Vector2(-speed, 0);
+        //    screenBounds = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, Camera.main.transform.position.z));
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        movement = new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical"));
+        if (Input.GetAxis("Horizontal") > 0)
+        {
+            transform.position += new Vector3(speed * Time.deltaTime, 0);
+        }
+        if (Input.GetAxis("Horizontal") < 0)
+        {
+            transform.position += new Vector3(-speed * Time.deltaTime, 0);
+        }
+        if (Input.GetAxis("Vertical") > 0)
+        {
+            transform.position += new Vector3(0,speed * Time.deltaTime);
+        }
+        if (Input.GetAxis("Vertical") < 0)
+        {
+            transform.position += new Vector3(0,-speed * Time.deltaTime);
+        }
+
+        float rightEdge = 10f; //the right edge of the screen in world coordinates
```

Comment:
I would move these to the top of the class declaration instead of here in the Update()